src/multiplayer/connect.js

```diff
@@ -1,8 +1,13 @@
-import { Client } from '@heroiclabs/nakama-js/dist/nakama-js.esm';
+import { Client } from '@heroiclabs/nakama-js';
```

Comment:
Hmm, interesting.

Reply:
It resolves to the same file, just feels safer this way.

Reply:
Yeah, it's nice. Back when we started using Nakama, the JS version was super broken and unmaintained overall, so we bumped into so many issues with it. Their NPM package was exploding in popularity, though the devs were kinda "meh, patches welcome".